contrib/babelfishpg_tsql/sql/upgrades/babelfishpg_tsql--1.2.0--1.3.0.sql

```diff
@@ -898,21 +898,221 @@ LANGUAGE SQL IMMUTABLE PARALLEL RESTRICTED;
 -- with different input/output data types. The second argument will be ignored as its functionality is not supported
```

Comment:
The changes should be moved to 2.0.0-2.1.0 upgrade script.

Reply:
Thank you, done. Now I need a maintainer to approve running workflows & 2 people to approve the PR.